src/app/personside/kontrollsporsmal/Kontrollsporsmal.tsx

```diff
@@ -68,9 +64,7 @@ function Kontrollsporsmal(props: Props) {
     return (
         <IfFeatureToggleOn toggleID={FeatureToggles.Kontrollspørsmål}>
             <KontrollSporsmalStyling role="region" aria-label="Visittkort-hode">
-                <div className="tittel">
-                    <Undertittel>Kontrollspørsmål</Undertittel>
-                </div>
+                <VisuallyHiddenAutoFokusHeader tittel="Kontrollspørsmål" />
```

Comment:
Ikke bruk denne her, ganske kritisk for skjermleser som nå mister fokus på navn og på sikkerhetstiltak hvis det finnes